Mono.Debugging/Mono.Debugging.Evaluation/AsyncEvaluationTracker.cs

```diff
@@ -41,7 +41,7 @@ namespace Mono.Debugging.Evaluation
 	/// will then be made asynchronous and the Run method will immediately return an ObjectValue
 	/// with the Evaluating state.
 	/// </summary>
-	public class AsyncEvaluationTracker: RemoteFrameObject, IObjectValueUpdater, IDisposable
```

Comment:
The library is designed so that the debugging engine can run in a remote process, which communicates with the client using remoting. That's really up to the debugger engine implementation. In such scenario, the AsyncEvaluationTracker could be executing in the remote process and generating ObjectValue instances that are serialized into the client process and which may need to hold a reference to that remote object (https://github.com/mono/debugger-libs/blob/ccadee54b48bcf932b8d8e69176e2ee56f47886b/Mono.Debugging/Mono.Debugging.Evaluation/AsyncEvaluationTracker.cs#L85). This model was necessary with the old Mono debugger, but it is not with the soft debugger. However, the infrastructure is still there, in case a specific debugger implementation needs it.